util/wordlists.js

```diff
@@ -6,12 +6,13 @@ var log = console.log
 var WORDLISTS = [
   'chinese_simplified',
   'chinese_traditional',
+  'czech',
   'english',
   'french',
   'italian',
   'japanese',
-  'spanish',
-  'korean'
+  'korean',
```

Comment:
Why did you change this order?

Reply:
Everything besides korean was sorted alphabetically, so I just corrected the order.